doc/source/basics.rst

```diff
@@ -1989,7 +2015,7 @@ force some *upcasting*.
 
 .. ipython:: python
 
-   df3.values.dtype
+   df3.to_numpy().dtype
```

Comment:
A few lines above this, the text still mentions `values` which should be `to_numpy` now to follow the example.